src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -2,8 +2,37 @@ export type DbCollection = {
     microorganism: string;
     samplingContext: string;
     matrix: string;
+    federalState: string;
+    samplingStage: string;
+    origin: string;
+    category: string;
+    productionType: string;

```

Comment:
could be: `export type DBCollection = Record<DbKey, string>;`

Reply:
It's done in the commit "Implement resistance-filter ".